lib/modules/manager/mint/extract.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { extractPackageFile } from '.';
+
+const simpleMintfile = Fixtures.get('simple.Mintfile');
+const noVersionMintfile = Fixtures.get('noVersion.Mintfile');
+
+const complexMintFileContent = `
+SwiftGen/SwiftGen@6.6.1
+yonaskolb/xcodegen
+realm/SwiftLint @ 0.48.0`;
+
+const includesCommentOutMintFileContent = `
+SwiftGen/SwiftGen@6.6.1
+yonaskolb/xcodegen
+#yonaskolb/xcodegen
+#realm/SwiftLint@0.48.0#commented out
```

Comment:
```suggestion
realm/SwiftLint@0.48.0 #commented out
```
looks like a better test